src/storage/src/table/cell_based_table.rs

```diff
@@ -51,7 +52,7 @@ pub struct CellBasedTable<S: StateStore> {
     column_descs: Vec<ColumnDesc>,
 
     /// Mapping from column id to column index
-    pk_serializer: Option<OrderedRowSerializer>,
+    pub pk_serializer: Option<OrderedRowSerializer>,
```

Comment:
why pub? is it used anywhere else?

Reply:
Yes, we need to `serialize_pk()` in StateTable.insert_row()/delete_row() since the relational interfaces accept a `Row`.